packages/cli/src/platform/logout.ts

```diff
@@ -8,8 +8,9 @@ export class Logout implements Command {
   }
 
   public async parse() {
-    const { token } = await readAuthConfig()
-    if (!token) {
+    const authJson = await readAuthConfig()
+    if (isError(authJson)) throw authJson
```

Comment:
I don't understand why this should throw here in logout 🤔 
It would better to delete the config instead I think